packages/notification-service/package.json

```diff
@@ -11,12 +11,13 @@
   "scripts": {
     "build": "tsc",
     "gcp-build": "npm run build",
-    "test": "",
+    "test": "jest",
     "lint": "tslint -c tslint.json --project tsconfig.json",
     "lint-checks": "yarn run lint && yarn build --noEmit",
     "start": "node ./dist/index.js",
     "start:local": "npm run build; cp config/config.local.env .env; node --inspect ./dist/index.js",
-    "deploy": "./deploy.sh"
+    "deploy": "./deploy.sh",
+    "test:watch": "export TZ=UTC && jest --watch"
```

Comment:
Please move this up to line 15.
Also suggest setting the TX on the `test` job and then making this line `yarn test --watch`